packages/hathor-rpc-handler/src/rpcMethods/sendNanoContractTx.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { HathorWallet } from '@hathor/wallet-lib';
+import {
+  TriggerTypes,
+  PinConfirmationPrompt,
+  PinRequestResponse,
+  TriggerHandler,
+  RequestMetadata,
+  SendNanoContractRpcRequest,
+  SendNanoContractTxConfirmationPrompt,
+  SendNanoContractTxConfirmationResponse,
+  SendNanoContractTxLoadingTrigger,
+  RpcResponseTypes,
+  RpcResponse,
+} from '../types';
+import { PromptRejectedError, SendNanoContractTxFailure } from '../errors';
+
+/**
+ * Sends a nano contract transaction.
+ *
+ * This function prompts the user for a PIN code and address, then uses these 
+ * to create and send a nano contract transaction based on the provided parameters.
+ *
+ * @param rpcRequest - The RPC request containing transaction details.
+ * @param wallet - The wallet instance to create/send the transaction.
+ * @param triggerHandler - The handler to manage user prompts.
+ *
+ * @returns The response from the transaction.
+ *
+ * @throws {SendNanoContractTxFailure} - If the transaction fails.
+ */
+export async function sendNanoContractTx(
+  rpcRequest: SendNanoContractRpcRequest,
+  wallet: HathorWallet,
+  requestMetadata: RequestMetadata,
+  triggerHandler: TriggerHandler,
+) {
+  const {
+    method,
+    blueprint_id,
+    nc_id,
+    actions,
+    args,
+    push_tx,
+  } = rpcRequest.params
+
+  const blueprintId = method === 'initialize' ? blueprint_id : nc_id;
```

Comment:
This is wrong. The `blueprintId` is always `blueprint_id`. My comment was regarding the data sent to the lib method.